src/extension.ts

```diff
@@ -102,6 +102,16 @@ export function activate(context: ExtensionContext) {
         } else {
           // The application.properties file belong to a Quarkus project, force to the quarkus-properties language
           languages.setTextDocumentLanguage(document, languageId);
+          const CONFIGURE_IN_SETTINGS = "Configure";
```

Comment:
you want an option to not show it again

Reply:
I personally don't think that it's worthwhile to do this. The notification only appears when they try to manually change the language ID off of Quarkus properties:

![LanguageIdUpdatedNotification](https://user-images.githubusercontent.com/22376627/88086245-0c76ea80-cb55-11ea-94e4-c40dfce836ff.gif)

After they have been notified once that what they just did had no effect, the user would either change the setting to disable the automatic updating (which means they will never get the notification again) or not try the action again, since they know it will do nothing. As long as they don't try the action again, the notification shouldn't pop up again.

Reply:
I added a "Disable Language Updating" button to the dialogue and changed the configure button to "Configure in Settings"